src/DynamoCoreWpf/Views/Preview/WatchTree.xaml

```diff
@@ -158,7 +160,45 @@
                                     <Setter Property="Visibility"
                                             TargetName="Expander"
                                             Value="Collapsed" />
+                                    <Setter Property="Margin"
+                                            TargetName="TreeViewItem"
+                                            Value="19,0,0,0" />
                                 </Trigger>
+
+                                <!-- __________________MultiDataTriggers______________________ -->
+                                <MultiDataTrigger>
```

Comment:
Trigger is for such situation:
![image](https://cloud.githubusercontent.com/assets/8158404/11561110/c90e429c-99cd-11e5-9534-9ae7e993511f.png)